examples/interop/ios-swiftui-in-compose/iosApp/iosApp/iosApp.swift

```diff
@@ -6,7 +6,7 @@ import UIKit
 struct iOSApp: App {
 	var body: some Scene {
 		WindowGroup {
-            ComposeViewControllerToSwiftUI()
+            ComposeViewControllerRepresentable()
```

Comment:
It looks only this line in the file uses spaces (other use tabs)

Reply:
We should use spaces everywhere to be consistent

Reply:
I will change all swift code tabs to spaces in separate PR

Reply:
made it in this PR: https://github.com/JetBrains/compose-multiplatform/pull/3609